ddl/index_change_test.go

```diff
@@ -103,9 +105,20 @@ func (s *testIndexChangeSuite) TestIndexChange(c *C) {
 			if err != nil {
 				checkErr = errors.Trace(err)
 			}
+		case model.StateWriteReorganization:
+			writeReorgTable, err1 := getCurrentTable(d, s.dbInfo.ID, tblInfo.ID)
+			if err1 != nil {
+				checkErr = errors.Trace(err1)
+			}
+			err = s.addRecordWriteReorg(ctx1, writeReorgTable, writeReorgStartVal, writeReorgStep)
+			if err != nil {
+				checkErr = errors.Trace(err)
+			}
+			writeReorgStartVal += writeReorgStep
 		case model.StatePublic:
-			if job.GetRowCount() != 3 {
-				checkErr = errors.Errorf("job's row count %d != 3", job.GetRowCount())
+			reorgRows := 3 + int64(writeReorgStep)
```

Comment:
Why row count is not 3?
If only 3 records exists before reorg, and use snapshot, job row count should be 3.